libmscore/cursor.cpp

```diff
@@ -53,22 +53,31 @@ void Cursor::setScore(Score* s)
 
 void Cursor::rewind(int type)
       {
-      if (type == 0) {
-            _segment = 0;
-            Measure* m = _score->firstMeasure();
-            if (m) {
-                  _segment = m->first(Segment::Type::ChordRest);
-                  firstChordRestInTrack();
-                  }
-            }
-      else if (type == 1) {
-            _segment  = _score->selection().startSegment();
-            _track    = _score->selection().staffStart() * VOICES;
-            firstChordRestInTrack();
-            }
-      else if (type == 2) {
-            _segment  = _score->selection().endSegment();
-            _track    = (_score->selection().staffEnd() * VOICES) - 1;  // be sure _track exists
+      switch(type) {
+            case 0:                 // score start of current track
+                  _segment = _score->firstSegment();
```

Comment:
How is that the start of current track? If track==1, segment will be empty. It's quite a big change.